embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/telemetry/TelemetryService.kt

```diff
@@ -9,6 +9,7 @@ internal interface TelemetryService {
 
     /**
      * Returns a map with every telemetry value. This is called when the session ends.
+     * We clear the usage count map so we don't count the same usages in the next session.
      */
     fun getTelemetryAttributes(): Map<String, String>
```

Comment:
Sorry - I should have been more specific. I think the NAME of the method should reflect that it will be destructive and not just a regular getting.